types/node/console.d.ts

```diff
@@ -150,11 +172,11 @@ declare module "node:console" {
              */
             debug(message?: any, ...optionalParams: any[]): void;
             /**
-             * Uses `util.inspect()` on `obj` and prints the resulting string to `stdout`.
+             * Uses [`util.inspect()`](https://nodejs.org/docs/latest-v20.x/api/util.html#utilinspectobject-options) on `obj` and prints the resulting string to `stdout`.
              * This function bypasses any custom `inspect()` function defined on `obj`.
              * @since v0.1.101
              */
-            dir(obj: any, options?: InspectOptions): void;
+            dir(obj: any, options?: ConsoleDirOptions): void;
```

Comment:
By the comment (Uses `util.inspect()` ...) and by the code (https://github.com/nodejs/node/blob/025ba26e517218bc744f580a7fbfa4bdebec0d0a/lib/internal/console/constructor.js#L388-L394) I see that options for `dir` the same as options for `util.inspect`. I don't think that this change is needed.